scale/mesos_api/tasks.py

```diff
@@ -103,6 +104,7 @@ def _create_docker_task(task):
     arguments = task.command_arguments.split(" ")
     for argument in arguments:
         mesos_task.command.arguments.append(argument)
+    mesos_task.command.arguments.append(settings.CONFIG_URI)
```

Comment:
It's unclear if this will work with an http, etc. URI since it only talks about file URIs in the documentation. Files aren't an option since the users aren't generally going to be able to place the config on each node (especially in aws type environments where nodes may come and go at any time).